css/components/download-item.postcss

```diff
@@ -0,0 +1,28 @@
+/*----------------------------------------*\
+  Download Item
+\*----------------------------------------*/
+
+.download-item {
+  @apply flex pl-0 pb-4 pt-4 align-top border-b border-gray-200 text-left w-full;
+  @apply hover:bg-secondary-100;
+}
+
+.download-item-icon {
+  @apply text-red-600 pr-6;
```

Comment:
adapting for mobile and tablets with `pr-2 md:pr-4 xl:pr-6` would be great

Reply:
true, done